store/precomputed_key/wvm/rpc/wvm_rpc_client.go

```diff
@@ -0,0 +1,241 @@
+package wvm
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	wvmtypes "github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/wvm/types"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+type Signer interface {
+	GetAccount(ctx context.Context) (common.Address, error)
+	SignTransaction(ctx context.Context, signData *wvmtypes.SignData) (string, error)
+}
+
+// WVM RPC client
+type RPCClient struct {
+	log     log.Logger
+	client  *ethclient.Client
+	chainID int64
+	signer  Signer
+}
+
+func NewWvmRPCClient(log log.Logger, cfg *wvmtypes.Config, signer Signer) (*RPCClient, error) {
+	client, err := ethclient.Dial(cfg.Endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to the WVM client: %w", err)
+	}
+
+	ethRPCClient := &RPCClient{
+		log:     log,
+		client:  client,
+		chainID: cfg.ChainID,
+		signer:  signer,
+	}
+
+	return ethRPCClient, nil
+}
+
+func (rpc *RPCClient) SendTransaction(ctx context.Context, to string, data []byte) (string, error) {
```

Comment:
just to verify - is there a maximum data size that can be submitted per tx?

Reply:
done, mentioned